leakpro/attacks/utils/shadow_model_handler.py

```diff
@@ -118,9 +117,18 @@ def create_shadow_models(
         if num_models < 0:
             raise ValueError("Number of models cannot be negative")
 
+        # Get shadow model class
+        if self.handler.configs["target"]["model_class"] is None:
+            logger.warning(
+                "Using the same model class for shadow models as the target model."
+            )
+            shadow_model_type = self.handler.configs["target"]["model_class"]
+        else:
+            shadow_model_type = self.handler.configs["shadow_model"]["model_class"]
```

Comment:
I dont understand this snippet. If the shadow_model does not have "model_class" in the config, this will crash.